hack/lib/common.bash

```diff
@@ -1,5 +1,7 @@
 #!/usr/bin/env bash
 
+# shellcheck disable=SC2001
```

Comment:
Can we define this locally to the line that violates it?

Reply:
ok.